client/lib/plugins/allowed-tags.js

```diff
@@ -0,0 +1,43 @@
+export const customTags = {
+	YOUTUBE: Symbol( 'youtube' ),
+};
+
+export const allowedTags = {
```

Comment:
Do you have any idea how this list came to be? I see it's adapting the pre-existing one that was used to sanitize HTML but was curious if you have any insights on how it was created and whether it needs some love.

Should we get it closer to https://github.com/WordPress/wordpress-develop/blob/6.0.2/src/wp-includes/kses.php#L61? 

I definitely don't want to expand the PR further, but I wanted to get your thoughts.

Reply:
Other than the discussion for the [initial PR](https://github.com/Automattic/wp-calypso/pull/19316), which added these rules, I'm not too familiar with the reasoning and many of the details and types of content that can pass through this.
With this in mind, I'd probably avoid making sweeping changes to this and add tags and attributes on a limited, per-need basis.